src/Stache/Stores/AggregateStore.php

```diff
@@ -14,6 +14,15 @@ public function __construct()
         $this->stores = collect();
     }
 
+    protected function resolveFromIndex($keys, $column)
+    {
+        return $this->stores()->mapWithKeys(function ($store) use ($column) {
+            return $store->resolveIndex($column)->load()->items();
+        })->where(function ($value, $key) use (&$keys) {
```

Comment:
for my learning, why is this `&$keys`? `$keys` is not modified in here.

Reply:
Tired brain. Had a non-collection array in there at first. Will do some cleanup this week; good catch 👍